frontend/app/src/components/Avatar.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
+    import { trackedEffect } from "@src/utils/effects.svelte";
     import { AvatarSize, OpenChat, UserStatus } from "openchat-client";
+    import { getContext } from "svelte";
     import Robot from "svelte-material-icons/Robot.svelte";
     import { rtlStore } from "../stores/rtl";
-    import { getContext } from "svelte";
     import { now } from "../stores/time";
-    import Verified from "./icons/Verified.svelte";
-    import { trackedEffect } from "@src/utils/effects.svelte";
+    import LighteningBolt from "./home/nav/LighteningBolt.svelte";
 
     const client = getContext<OpenChat>("client");
 
-    type Specialisation = "verified" | "bot" | "none";
+    type Specialisation = "bot" | "none" | "super_user";
```

Comment:
Maybe rather than "super_user" we say "max_streak"?
I think that will result in less confusion when people see this value.